### PR TITLE
feat: added UI for sticky notes

### PR DESCRIPTION
**Pull request by aryan-13**
Added:
- Routes for tasklist and notes
- Input field for notes
- Notes component

image

### CODE REVIEW

File: src/Components/MainSection/Notes/Notes.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	padding: 1rem 0 2rem 0;
+
+	width: 60rem;
+	text-align: center;
+	font-size: 2.5rem;
+}
+.note-box {
+	display: flex;
+	max-height: 20rem;
+	/* width: 25rem; */
```
Comment (anshikaa12) [inline src/Components/MainSection/Notes/Notes.css]: remove commented code

File: src/App.jsx [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import './App.css';
 import './Components/Header/Header.css';
+import { Routes, Route } from 'react-router-dom';
 import TaskList from './Components/MainSection/TaskList/TaskList';
```
Comment (anshikaa12) [inline src/App.jsx:4]: create an index.js in components folder and import all from there

File: src/Components/MainSection/Notes/Notes.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
+.notes-container {
+	padding: 1rem 2rem;
+	height: 85vh;
+}
+.notes-heading {
+	padding: 1rem 0 2rem 0;
+
```
Comment (anshikaa12) [inline src/Components/MainSection/Notes/Notes.css]: you can remove extra spaces and use prettier

**Review by anshikaa12 — APPROVED** [review]
Clean UI
Clean code!

File: src/Components/MainSection/Notes/Notes.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+	resize: none;
+}
+.btn-notes {
+	border-top-left-radius: 0;
+	border-top-right-radius: 0;
+}
+
+.user-notes {
+	overflow-y: auto;
+	height: 46rem;
```
Comment (lannisterrr) [inline src/Components/MainSection/Notes/Notes.css]: try not to give fixed height , use min-height instead.

**Review by lannisterrr — COMMENTED** [review]
Code is structured , some suggesstions made.

File: src/Components/MainSection/Notes/Notes.css (hunk 1 above)
Comment (aryan-13) [reply src/Components/MainSection/Notes/Notes.css]: done

File: src/Components/MainSection/Notes/Notes.css (hunk 3 above)
Comment (aryan-13) [reply src/Components/MainSection/Notes/Notes.css]: done

File: src/Components/MainSection/Notes/Notes.css (hunk 4 above)
Comment (aryan-13) [reply src/Components/MainSection/Notes/Notes.css]: fixed

**Pull request merged**